### PR TITLE
add additional files to release bundle

### PR DESCRIPTION
**Pull request by GregHanson**
currently only the istio.VERSION and `*.yaml` files are getting added to the release.  This will copy all current and future `cleanup*` scripts

Added root level LICENSE and README's to release as well

---
This change is [Reviewable](https://reviewable.io/reviews/istio/istio/290)

### CODE REVIEW

**Comment by GregHanson** [comment]
resolves https://github.com/istio/istio/issues/289

**Comment by ijsnellf** [comment]
@sebastienvas

File: release/create_release_archives.sh [hunk 1]
```diff
@@ -76,7 +76,7 @@ function create_windows_archive() {
 
 pushd ${ROOT}
 cp istio.VERSION "${COMMON_FILES_DIR}"/
-find samples install -type f -name "*.yaml" \
+find samples install -type f -name "*.yaml" -o -name "cleanup*" \
```
Comment (sebastienvas) [inline release/create_release_archives.sh]: Just ran this, this does not work, you need to do:
\\( -name "*.yaml" -o -name "cleanup*" \\)

also when I ran with my change the only file added is:
samples/apps/bookinfo/cleanup.sh

Is that what you expect ?

Comment (GregHanson) [inline release/create_release_archives.sh]: yes, there is currently only one cleanup script but wanted to leave it open for any others that may be added

**Pull request merged**